packages/lpa-questionnaire-web-app/tests/unit/controllers/authentication.test.js

```diff
@@ -0,0 +1,93 @@
+const authenticationController = require('../../../src/controllers/authentication');
+const { mockReq, mockRes } = require('../mocks');
+const { VIEW } = require('../../../src/lib/views');
+
+const mockEmail = 'test.email@test.uk';
+const mockAppealId = '123';
+
+const res = mockRes();
+const req = mockReq({}, mockAppealId);
+
+describe('controllers /authentication/your-email', () => {
```

Comment:
I think these tests could be under one describe, with each test naming the endpoint.

Reply:
Yes, sounds good. I have updated them.